modules/cms/classes/ComponentBase.php

```diff
@@ -131,6 +137,11 @@ public function onRun()
      */
     public function onRender()
     {
+        if ($this->cacheTTL) {
+            return Cache::remember($this->getCacheKey(), now()->addMinutes($this->cacheTTL), function () {
```

Comment:
cacheTTL should also support being a datetime value in which case it should just be passed straight to Cache::remember